### PR TITLE
Lg20201111 my lemonade documentation

### PR DESCRIPTION
**Pull request by janainawittner**

### CODE REVIEW

File: content/apps/my-lemonade/en/overview.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+A Kitchen Display Screen is available to restaurants without EPOS integration to receive orders and update product availabilities.
+
+## Integration Features
+
+Connecting My Lemonade to HubRise allows My Lemonade to:
+
+- Push orders into HubRise.
+- Pull order status from HubRise into My Lemonade.
+- Push customer information into HubRise.
```
Comment (janainawittner) [inline content/apps/my-lemonade/en/overview.md:33]: I included **Pull order status from HubRise into My Lemonade** as I have seen that this is done in the integration sheet. I also included **Push customer information into HubRise** as all online ordering solutions do this. I haven't even checked the integration sheet :-)

If you have doubts that any of this is correct, do not hesitate to challenge it.

File: content/apps/my-lemonade/en/overview.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  category: Online ordering
+  availability: Worldwide
+  price_range: €19-€49 / month
+  website: https://mylemonade.app
+  contact: contact@mylemonade.app
+---
+
+## Description
+
+My Lemonade is a table ordering solution. Customers flash the QR code or use the NFC tag placed on their table to access the menu on their smartphone and order. Customer can choose to pay directly on the app or with the waiter. The menu displays in the customer's language and there is nothing to install.
```
Comment (janainawittner) [inline content/apps/my-lemonade/en/overview.md:21]: My understanding is that My Lemonade is a table ordering solution. You might also be able to order remotely but I am not even sure about this. We need to double check. 

Make a note in Trello.

Comment (janainawittner) [inline content/apps/my-lemonade/en/overview.md:21]: How does the language display work? Does it uses by default the language of the smartphone brower?
How many languages can be available?
What happens if the language is not available in the menu. 

Make a note in Trello.

File: content/apps/my-lemonade/en/overview.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+position: 1
+title: Overview
+layout: documentation
+meta:
+  title: My Lemonade Connection to HubRise - Overview
+  description: My Lemonade overview and why you should connect it to HubRise. With a connection to HubRise orders are sent to tools you use every day including your EPOS.
+gallery:
+  - 003-fr-2x-my-lemonade-sample-app.png
```
Comment (janainawittner) [inline content/apps/my-lemonade/en/overview.md:9]: We cannot use the three images one beside another. it does not look good in context. 
I have asked the client for more illustrations. Also asked for FR and EN if possible.
I alreadt made a note about this in Trello.

**Comment by janainawittner** [comment]
@LeGaby, it's all yours :-)

File: content/apps/my-lemonade/en/overview.md (hunk 2 above)
Comment (LeGaby) [reply content/apps/my-lemonade/en/overview.md:21]: @janainawittner, I don't see any reference to a possibility to order remotely. What do you mean by that?

Comment (janainawittner) [reply content/apps/my-lemonade/en/overview.md:21]: I think My Lemonade is an order at the table solution. I don't think it works like an online ordering solution. I don't think you can order if you are not at the restaurant....

Comment (LeGaby) [reply content/apps/my-lemonade/en/overview.md:21]: I think you are right, I forgot I had put that in the beginning and I didn't see the reference. Good you removed it!

Comment (janainawittner) [reply content/apps/my-lemonade/en/overview.md:21]: I guess you can close this conversation then :-)

**Comment by janainawittner** [comment]
@LeGaby are we ready to merge this ?

**Comment by LeGaby** [comment]
@janainawittner, I think we are!

**Comment by antmonnier** [comment]
Good job Leo! Please assign to me and I'll merge it to master then.

**Comment by janainawittner** [comment]
You guys will have to explain to me how Antoine picked this up....
This PR has not been assigned to you @antmonnier, and you are not mentioned anywhere that I can see.... 
How is this even possible?
Just trying to understand how this thing really works.

**Pull request merged**